subsys/fs/fuse_fs_access.c

```diff
@@ -468,6 +469,12 @@ static void fuse_fs_access_init(void)
 {
 	int err;
 	struct stat st;
+	int i = 0;
```

Comment:
Better to have this be `size_t` since it can never be negative and it's being compared to a value of type `size_t`.